src/lib/constants.js

```diff
@@ -137,5 +137,8 @@ export const METADATA = {
     TEMPLATE_AUTOCAD: 'autocad',
 };
 
+export const BROWSER_EDGE = 'Edge';
```

Comment:
Do these really _need_ to be constants? If so, can we replace any other hardcoded usages of the strings, as well? Can we namespace the constant, like so?

```
const BROWSERS = {
    EDGE: 'Edge',
    EXPLORER: 'Explorer,
};
```